.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 .idea
 /vendor
+
+docker-container/docker-magento
```

Comment:
You'll want to add `config.sh` and `auth.json`.

Reply:
Added the config.sh, but there is no auth.json for M1